js/error-message.js

```diff
@@ -0,0 +1,76 @@
+import { isEscEvent } from './utils.js';
+
+const ALERT_SHOW_TIME = 8000;
+
+const errorTemplate = document.querySelector('#error').content.querySelector('.error');
+const errorPopup = errorTemplate.cloneNode(true);
+const closeErrorPopupButton = errorPopup.querySelector('.error__button');
+const errorContainer = errorPopup.querySelector('.error__inner');
+
+const showErrorGetData = () => {
+  const alertContainer = document.createElement('div');
+  alertContainer.style.zIndex = 100;
```

Comment:
```suggestion
  alertContainer.style.zIndex = '100';
```

style принимает только стринги, апи у него такой. (что, кстати, раздражает :), тоже хочется воткнуть в него цифры)

Reply:
поправь остальные намберы

Reply:
Для информации -  https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration это вот это апи